css/master.css

```diff
@@ -0,0 +1,229 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  width: 100%;
+  height: 600px;
+  background-image: url("https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg");
+  background-position: bottom;
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-origin: border-box;
+  display: flex;
+  flex-direction: column;
+}
+
+header nav {
+  width: 85%;
+  height: 50px;
+  margin: 100px auto 50px auto;
+  display: flex;
+  justify-content: space-between;
+  border-radius: 4px;
+  box-shadow: 0 0 4px black;
+  background: rgb(210, 210, 242);
+  background:
+    linear-gradient(
+      0deg,
+      rgba(210, 210, 242, 1) 0%,
+      rgba(221, 210, 184, 1) 51%,
+      rgba(171, 153, 144, 1) 57%,
+      rgba(124, 94, 94, 1) 100%
+    );
+}
+
+.vl {
+  border-left: 1px solid lightgray;
+  margin-left: 5px;
+  padding-left: 5px;
+}
+
+nav .vl {
+  border-left: 1px solid rgb(98, 98, 98);
+  margin-left: 0;
+  padding-left: 0;
+}
+
+header nav div {
+  width: 100%;
+}
+
+header nav > div:hover {
+  background: rgb(210, 210, 242);
+  background:
+    linear-gradient(
+      0deg,
+      rgba(170, 170, 202, 1) 0%,
+      rgba(181, 170, 144, 1) 51%,
+      rgba(131, 103, 104, 1) 57%,
+      rgba(84, 54, 54, 1) 100%
+    );
+  color: white;
+}
+
+header nav button,
+.search-icon-container {
+  width: 100%;
+  border-radius: 0;
+  border: none;
+  background-color: white;
+  background-color: rgba(0, 0, 0, 0);
+}
+
+header nav button:hover {
+  background: rgb(210, 210, 242);
+  background:
+    linear-gradient(
+      0deg,
+      rgba(170, 170, 202, 1) 0%,
+      rgba(181, 170, 144, 1) 51%,
+      rgba(131, 103, 104, 1) 57%,
+      rgba(84, 54, 54, 1) 100%
+    );
+  color: white;
+}
```

Comment:
`[Optional but nice to have]` It would be nice to use colors near the original website for your clone, you can use developer tools to get the colors.

Reply:
Thank you for all feedback, i know it is meant to be a clone, but those grayish colors are so... boring. I did changed the background but to a better collor, i hope you like it !

Thanks for all your feedback !